modules/host/negotiatesettings.go

```diff
@@ -26,54 +26,60 @@ func (h *Host) capacity() (total, remaining uint64, err error) {
 	return total, remaining, nil
 }
 
+// externalSettings compiles and returns the external settings for the host.
+func (h *Host) externalSettings() (modules.HostExternalSettings, error) {
+	totalStorage, remainingStorage, err := h.capacity()
+	if err != nil {
+		return modules.HostExternalSettings{}, err
+	}
```

Comment:
hmm, seems strange that returning the external settings can fail. I feel like it should be possible to track capacity without needing to read from disk every time.